overlay.yml

```diff
@@ -32,3 +32,6 @@ components:
     distgit:
         name: containernetworking-cni
         patches: drop
+
+  - src: github:cgwalters/redhat-release-coreos
```

Comment:
We should obviously figure out hosting for this... @smarterclayton WDYT about `github.com/openshift/redhat-release-coreos` ?

Reply:
WFM

Reply:
WFM - transfer it to me and i'll set up permissions